Orange/widgets/tests/base.py

```diff
@@ -176,7 +176,119 @@ def get_output(self, output_name, widget=None):
         return self.signal_manager.outputs.get((widget, output_name), None)
 
 
-GuiToParam = namedtuple('GuiParam', 'name gui_el get set values set_values')
+class BaseParameterMapping:
+    """Base class for mapping between gui components and learner's parameters
+    when testing learner widgets.
+
+    Parameters
+    ----------
+    name : str
+        Name of learner's parameter.
+
+    gui_element : QWidget
+        Gui component who's corresponding parameter is to be tested.
+
+    values: list
+        List of values to be tested.
+
+    getter: function
+        It gets component's value.
+
+    setter: function
+        It sets component's value.
+    """
+
+    def __init__(self, name, gui_element, values, getter, setter):
+        self.name = name
+        self.gui_element = gui_element
+        self.values = values
+        self.get_value = getter
+        self.set_value = setter
+
+
+class DefaultParameterMapping(BaseParameterMapping):
+    """Class for mapping between gui components and learner's parameters
+    when testing unchecked properties and therefore default parameters
+    should be used.
+
+    Parameters
+    ----------
+    name : str
+        Name of learner's parameter.
+
+    gui_element : QWidget
```

Comment:
gui_element was removed from DefaultParameterMapping so this is considered fixed.